### PR TITLE
DEMO-46: Use symfony command during install

### PR DESCRIPTION
**Pull request by nicolas-bastien**
- Use doctrine command to create the database directly in bash
- rename default database name to fit the project and remove confilt with ezplatform installation

https://jira.ez.no/browse/DEMO-46

### CODE REVIEW

**Comment by andrerom** [comment]
where does `doctrine:database:create` take schema from?
Could probably sync this up with work from https://github.com/ezsystems/ezpublish-kernel/pull/1794

I would be interested in disucssion thins, afaik what we need is a migrations command to replace the install one as well as provide migrations on changes, this could be doctrine migrations, over time sprinkled with eZ features for api migrations with features found in the misc eZ Migration Bundles out there.

Why: It would let us simplify setup on platform.sh and docker, so same command can safely be executed when container starts no matter if clean install or migration to new version of app container.

File: README.md [hunk 1]
```diff
@@ -45,6 +37,14 @@ php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatfor
 
 *Installation will ask you for database credentials and secret token for Symfony, other settings can stay as default.*
 
+### 2. Create a database
+
+Then, create a new database using the following command:
+
+```bash
+php app/console doctrine:database:create
```
Comment (bdunogier) [inline README.md]: This could actually apply to `ezplatform` as well.

**Comment by nicolas-bastien** [comment]
@andrerom I agree there is a conceptual pb here.

As the repo is responsible for storing the data, installation task should be done in the repo and no sql should be done in the project.

How do you want to work on that ?

**Comment by nicolas-bastien** [comment]
@clash82 link to your jira in commit done

**Pull request merged**